juju/juju
# snap: Pull operator images if microk8s is installed

## Description of change

Adds hooks for install and refresh (upgrade etc.) that will pull the operator
images if microk8s is installed.

Drive-by cleanup of the snapcraft.yaml too.

## QA steps

Manual steps done:
  - Create the snap:
    - run ```snapcraft``` in juju/juju root
  - have microk8s installed
    - run ```sudo snap install microk8s --classic```
  - Check current images:
    - run ```microk8s.ctr images ls```
  - Install the snap:
    - run ```sudo snap install --dangerous --classic ./<path to snap>.snap
  - Check for new images (this should have 2, jujud-operator and juju-db):
    - run ```microk8s.ctr images ls```

## Documentation changes

n/a

## Bug reference

n/a
